pkg/models/licensing.go

```diff
@@ -14,8 +14,6 @@ type Licensing interface {
 
 	StateInfo() string
 
-	EnabledFeatures() map[string]bool
```

Comment:
@DanCech -- I removed the enumeration from the main interface.  Is the map really necessary?

Reply:
@DanCech -- do licenses ever set things to *false*?

Reply:
removing it is fine I think but we'll need to verify in enterprise, the map was intended to just be a list of true values, no false values.